### ISSUE TITLE
[Package]: Neovim 0.8

### ISSUE THREAD

**Issue by pandas-id** [termux/termux-packages#11189]
### Package description

I hope neovim 0.8 can already be installed in termux

### Home page URL

https://neovim.io/

### Source code URL

https://github.com/neovim/neovim

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

**Comment by 2096779623** [comment]
Here is probably the best option for this package request: https://github.com/termux-user-repository/tur.

**Comment by Anonymous1231-dev** [comment]
Pls take note that neovim 0.8 is not stable yet. Pls use `neovim-nightly` maybe

**Comment by pandas-id** [comment]
I hope they can add the neovim-nightly package to termux-package, so that people can use it right away even though it's not stable yet.

**Comment by Grimler91** [comment]
> I hope they can add the neovim-nightly package to termux-package

We already have it:
```
$ apt search neovim-nightly
Sorting... Done
Full Text Search... Done
neovim-nightly/stable 0.8.0-dev+572-g6b72ee1f1-0 aarch64
  Ambitious Vim-fork focused on extensibility and agility (nvim-nightly)
```

**Comment by pandas-id** [comment]
I just found out that I can install it from there

**Comment by Dmole** [comment]
> We already have [neovim]

Next up [lunarvim](https://www.lunarvim.org/#opinionated) for that automatic LSP goodness.

**Comment by thunder-coding** [comment]
I guess this issue should be closed, Neovim 0.8 isn't stable, and also is currently available as neovim-nightly. Maybe can be reopened once 0.8 is pushed out as stable and we're lacking behind the version :)
